### DISCUSSION TITLE
how do I jump to next error?

### DISCUSSION THREAD

**Discussion by ghost** [LunarVim/LunarVim#4064] [Q&A] [answered]
I wish cursor could jump to the next error, so I tried `:cnext` command, but it showed "No Error**
How could I jump to the next error?

**Comment by brneor** [comment, marked as answer]
Not sure if that's what you want, but `<leader> l j` and `<leader> l k` navigate to next and previous LSP diagnostic respectively.

**Reply by cassepipe** [reply]
Hi @brneor,
I use those a lot and I  am used to map them to `<S-j>` and `<S-k>`.
Do you happen to know which commands I should map ? 
Not sure which of the `lvim.builtin` plugins is responsible for that

**Reply by cassepipe** [reply]
I was told on the Discord to add mappings for
```vimscript
<cmd>lua vim.diagnostic.goto_next({buffer=0})<CR>
<cmd>lua vim.diagnostic.goto_prev({buffer=0})<cr>
```
So not a builtin plugin
`:h vim.diagnostic`
```
Diagnostic framework                                     *vim.diagnostic*

Nvim provides a framework for displaying errors or warnings from external
tools, otherwise known as "diagnostics". These diagnostics can come from a
variety of sources, such as linters or LSP servers. The diagnostic framework
is an extension to existing error handling functionality such as the
|quickfix| list.

```
I wonder what the `{buffer=0}` is for. Probably means the current buffer...
